docs/install/README.md

```diff
@@ -5,7 +5,7 @@ The following is an overview of the different installation methods available.
 ## Prerequisites
 
 Kata Containers requires nested virtualization or bare metal. Check 
-[hardware requirements](/src/runtime/README.md#hardware-requirements) to see if your system is capable of running Kata 
+[hardware requirements](./../..//README.md#hardware-requirements) to see if your system is capable of running Kata 
```

Comment:
There is an extra slash here, but it still work at the moment
```suggestion
[hardware requirements](./../../README.md#hardware-requirements) to see if your system is capable of running Kata 
```

Reply:
updated.